examples/hotrod/README.md

```diff
@@ -48,7 +48,6 @@ Jaeger UI can be accessed at http://localhost:16686.
 ```bash
 go get github.com/jaegertracing/jaeger
 cd $GOPATH/src/github.com/jaegertracing/jaeger
```

Comment:
these two should probably be `git clone` and `cd jaeger`, rather than relying on GOPATH.